tests/TestUTestMacro.cpp

```diff
@@ -546,8 +546,8 @@ static void _failingTestMethodWithBYTES_EQUAL()
 TEST(UnitTestMacros, FailureWithBYTES_EQUAL)
 {
     fixture.runTestWithMethod(_failingTestMethodWithBYTES_EQUAL);
-    CHECK_TEST_FAILS_PROPER_WITH_TEXT("expected <97 0x61>");
-    CHECK_TEST_FAILS_PROPER_WITH_TEXT("but was  <98 0x62>");
+    CHECK_TEST_FAILS_PROPER_WITH_TEXT("expected <97 (0x61)>");
+    CHECK_TEST_FAILS_PROPER_WITH_TEXT("but was  <98 (0x62)>");
```

Comment:
I wanted to add it after yours was merged, but since @basvodde merged mine first, it was already there (mismerge?) and now yours is the privilege of fixing it up :-)